src/MixedLayerThermoclineDynamics.jl

```diff
@@ -2,25 +2,7 @@ module MixedLayerThermoclineDynamics
 
 export hello, domath
```

Comment:
```suggestion
export Grid1D, Grid2D
```

Reply:
I think these deserve to be exported.

Reply:
Yes, that's true. We should do this.

Reply:
Just click on the "Commit suggestion" button and it will be done! :)